### PR TITLE
chore: format all files

### PR DESCRIPTION
**Pull request by boneskull**
This just formats everything all at once, if we want to do that.

### CODE REVIEW

**Comment by boneskull** [comment]
If we do something like this, we can then enable something like what Endo does where a format check is a lint step.

**Comment by boneskull** [comment]
Current dependencies on/for this PR:
* main
  * **PR #725** <a href="https://app.graphite.dev/github/pr/LavaMoat/LavaMoat/725" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/LavaMoat/LavaMoat/725?utm_source=stack-comment).

**Review by leotm — APPROVED** [review]
lgtm locally running `npm -- exec prettier --write **/*.(ts|md|ya?ml)`

but can't run `.js` in recursive dirs :suspect: if any ideas

```console
npm -v # 9.8.1
npm -- exec prettier --write **/*.(js|ts|md|ya?ml) # zsh: argument list too long: npm
npm -- exec prettier --write **/*.js # zsh: argument list too long: npm
npm -- exec prettier --write *.js # ok
ulimit -s # 81xx
ulimit -s 65500 # try increasing ulimit
npm -- exec prettier --write **/*.js # zsh: argument list too long: npm
```

presume it's a Mac thing, so our `ubuntu-latest` machines should be ok (when we format check as lint step too)

**Comment by naugtur** [comment]
> npm -- exec prettier --write **/*.js # zsh: argument list too long: npm


This is a glob pattern and prettier uses glob. Is it not a matter of putting it in quotes? 
```
npm -- exec prettier --write "**/*.js"
```

File: packages/browserify/examples/00-simple-cli/xyz-bundle.js [hunk 1]
```diff
@@ -1,3 +1,49 @@
-(function(){function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){var n=e[i][1][r];return o(n||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}return r})()({1:[function(require,module,exports){
```
Comment (legobeat) [inline packages/browserify/examples/00-simple-cli/xyz-bundle.js:1]: I guess these minified files should be excluded

File: packages/node/examples/eval-server/lavamoat-config.json [hunk 2]
```diff
@@ -67,4 +67,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```
Comment (legobeat) [inline packages/node/examples/eval-server/lavamoat-config.json:70]: I think these will be reverted on next regeneration (which should prob be addressed)?

File: packages/viz/src/css/bootstrap.css [hunk 3]
```diff
@@ -3,5 +3,9250 @@
...
...e{font:0/0 a;color:transparent;text-shadow:none;background-color:transparent;border:0}.text-decoration-none{text-decoration:none!important}.text-break{word-break:break-word!important;overflow-wrap:break-word!important}.text-reset{color:inherit!important}.visible{visibility:visible!important}.invisible{visibility:hidden!important}@media print{*,::after,::before{text-shadow:none!important;box-shadow:none!important}a:not(.btn){text-decoration:underline}abbr[title]::after{content:" (" attr(title) ")"}pre{white-space:pre-wrap!important}blockquote,pre{border:1px solid #adb5bd;page-break-inside:avoid}thead{display:table-header-group}img,tr{page-break-inside:avoid}h2,h3,p{orphans:3;widows:3}h2,h3{page-break-after:avoid}@page{size:a3}body{min-width:992px!important}.container{min-width:992px!important}.navbar{display:none}.badge{border:1px solid #000}.table{border-collapse:collapse!important}.table td,.table th{background-color:#fff!important}.table-bordered td,.table-bordered th{border:1px solid #dee2e6!important}.table-dark{color:inherit}.table-dark tbody+tbody,.table-dark td,.table-dark th,.table-dark thead th{border-color:#dee2e6}.table .thead-dark th{color:inherit;border-color:#dee2e6}}
```
Comment (legobeat) [inline packages/viz/src/css/bootstrap.css]: This vendored file should prob be excluded as well?

File: packages/browserify/examples/00-simple-cli/index.html [hunk 4]
```diff
@@ -1,10 +1,10 @@
-<!DOCTYPE HTML>
```
Comment (naugtur) [inline packages/browserify/examples/00-simple-cli/index.html:1]: (could) examples are ignored by eslint and I think we should consider using the same ignore rules for prettier.

Comment (boneskull) [reply packages/browserify/examples/00-simple-cli/index.html:1]: I'll probably leave this as-is; I don't think it's harmful to format them.

File: packages/browserify/examples/00-simple-cli/xyz-bundle.js (hunk 1 above)
Comment (boneskull) [reply packages/browserify/examples/00-simple-cli/xyz-bundle.js:1]: I'll rename this file to .min.js or something.

**Comment by boneskull** [comment]
@leotm Just running `npm exec -- prettier --write .` works

File: packages/viz/src/css/bootstrap.css (hunk 3 above)
Comment (boneskull) [reply packages/viz/src/css/bootstrap.css]: vendored files should be clearly marked as vendored lol

File: packages/browserify/examples/00-simple-cli/xyz-bundle.js (hunk 1 above)
Comment (boneskull) [reply packages/browserify/examples/00-simple-cli/xyz-bundle.js:1]: it's unclear what the purpose of this file is.  `xyz.js` and `xyz-bundle.js` aren't referenced anywhere

Comment (boneskull) [reply packages/browserify/examples/00-simple-cli/xyz-bundle.js:1]: I've removed this and `xyz.js`

File: packages/viz/src/css/bootstrap.css (hunk 3 above)
Comment (boneskull) [reply packages/viz/src/css/bootstrap.css]: I've flagged this as ignored

**Pull request merged**